frontend/app/pages/welcome/welcome.tsx

```diff
@@ -5,6 +5,10 @@ import Header from "~/custom-components/header";
 import Layout from "~/custom-components/layout";
 import Text from "~/custom-components/text";
 
+export function loader() {
+  return null;
+}
+
 export function Welcome() {
```

Comment:
I think the react router issue is fixed like this
```suggestion
export default function Welcome() {
```

Reply:
not up to date anymore